### DISCUSSION TITLE
Workflow

### DISCUSSION THREAD

**Discussion by joaopauloalbq** [kraxarn/spotify-qt#245] [Ideas]
I think these three colored containers should appear just one at a time, as Spotify does.

When I going to look for an artist (blue), the current playlist no longer matters (yellow), when I find the artist (green), the list of artists searched (blue) no longer matters, and so on.

The _Audio Resource_ should be a dialog window IMO, since we will not be looking at this information all the time, unlike the lyrics.

![Screenshot_20200516_200031](https://user-images.githubusercontent.com/9905202/82131963-3278bb00-97b1-11ea-96bc-b9c73cc7c812.png)

**Comment by kraxarn** [comment]
I think my app makes better use of the space, as everything is more compact, than the official client and that's why it's designed like that. I do see your point though and I may do an option for it at some point. I do agree though that the audio features, if that is the one you meant, should be a dialog or something similar. It was mostly just thrown together as I just found it odd that I could even get that information from the API, but it also needs some rework, mostly as the values shown doesn't make much sense.
